scripts/test-crosschain.sh

```diff
@@ -4,7 +4,10 @@ yarn wait-on tcp:8000 &&
 wait-on tcp:8545 && 
 yarn hardhat test test/crosschain/ZodiacExecution.test.ts --network 'ethereumLocal' --starknet-network 'starknetLocal' && 
 sleep 10 && 
-yarn hardhat test test/crosschain/EthTxAuth.test.ts --network 'ethereumLocal' --starknet-network 'starknetLocal'
+yarn hardhat test test/crosschain/EthTxAuth.test.ts --network 'ethereumLocal' --starknet-network 'starknetLocal' && 
+sleep 10 &&
```

Comment:
why do we add a sleep here?

Reply:
some issues with the devnet, might be working now with the updates